### PR TITLE
Astro 1211 radio

### PR DESCRIPTION
**Pull request by Cheshire89**
https://rocketcom.atlassian.net/browse/ASTRO-1211

adjust radio button sizing and fix hover behavior
PR includes Duncans production fixes

### CODE REVIEW

**Review by dmcalester — CHANGES_REQUESTED** [review]
Two required changes and a couple of notes
- Update the hard coded width property with the var (inline comment)
- There are merge conflicts in the compiled CSS

There is a discrepancy in the size from design. Sketch library is showing 16px height/width, but **don’t change** I want to understand why the design change.

**Comment by amalyah** [comment]
Could you add a link to the relevant ticket in the description?

File: src/css/common/__variables.css [hunk 1]
```diff
@@ -8,7 +8,7 @@
   --buttonBorderRadius: 3px;
   --defaultBorderRadius: 3px;
 
-  --controlOptionSize: 1.25rem;
+  --controlOptionSize: 1.125rem; /* 18px */
```
[the remark this answers is not in the document]
Comment (amalyah) [reply src/css/common/__variables.css:11]: The latest version of the PI branch has the radio buttons as 18x18. @dmcalester, the link you have there is not the source for these change requests. That is an incomplete spike for structure changes to the symbol.

File: src/css/core/control.css [hunk 2]
```diff
@@ -52,6 +52,8 @@
   justify-content: flex-start;
 
   color: var(--controlLabelColor);
+  letter-spacing: 0.5px;
```
Comment (amalyah) [inline src/css/core/control.css:55]: The letter-spacing is not specific to these controls. That letter spacing should be true for all body-sized text.

File: static/css/astro.core.css [hunk 3]
```diff
@@ -1760,6 +1760,8 @@ label {
   justify-content: flex-start;
 
   color: var(--controlLabelColor);
+  letter-spacing: 0.5px;
```
[the remark this answers is not in the document]
Comment (amalyah) [reply static/css/astro.core.css:1763]: This letter-spacing is true for any body-sized font.

**Review by amalyah — APPROVED** [review]
Side Note: The Radio buttons readme page references the "Toggle and Push Buttons" UXDS page. We should update the link to separate the two topics and then change Toggle to Switch.

File: src/css/common/__variables.css (hunk 1 above)
[the remark this answers is not in the document]
Comment (Cheshire89) [reply src/css/common/__variables.css:11]: @dmcalester there was merge conflicts with minified astro.core and astro css files which might have caused it

**Review by cskerbo — APPROVED** [review]
No changes from me, but please be sure to include the link to the relevant ticket upon initial creation of future PRs. 

Thanks for your hard work!

https://rocketcom.atlassian.net/browse/ASTRO-1211

**Review by dmcalester — APPROVED** [review]

**Pull request merged**